README.md

```diff
@@ -29,17 +29,17 @@ Input latency is tested using the methodology outlined at [WydD's inputlag.scien
 
 | Version | Mode | Poll Rate | Min | Max | Avg | Stdev | % on time | %1f skip | %2f skip |
 | - | - | - | - | - | - | - | - | - | - |
-| v0.3.1 | All | 1 ms | 0.56 ms | 1.32 ms | 0.85 ms | 0.24 ms | 95.95% | 4.05% | 0% |
+| v0.4.3 | All | 1 ms | 0.46 ms | 1.24 ms | 0.77 ms | 0.24 ms | 96.37% | 3.63% | 0% |
```

Comment:
@FeralAI could we get numbers for v0.6.3 added here?

Reply:
These are the numbers from my latest test with XInput (should be pretty much same for all modes) on a fresh device with v0.6.3:


polling interval | 1.0
-- | --
min | 0.46
max | 1.64
average | 0.89
stddev | 0.29
% no skip | 95.59%
% skip 1 frame | 4.41%
% skip 2 frames | 0%
% skip 3 frames | 0%